source/quick-start.txt

```diff
@@ -37,20 +37,25 @@ to check your version of Node and npm, as well as installation
 instructions, see `downloading and installing Node.js and npm 
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
+Select the :guilabel:`Clone the Project` tab if you want to build your
+project from a premade dependency file or the :guilabel:`Create the Project`
```

Comment:
Comment: I've never heard a `package.json` referred to as a dependency file (not wrong, but just doesn't seem quite right to me). I googled around a bit this site refers to it as a "[manifest](https://nodejs.dev/learn/the-package-json-guide)" which sounds like a better descriptor to me (captures more than listing dependencies). Non-blocking either way.

```suggestion
project from a premade manifest file or the :guilabel:`Create the Project`
```

Reply:
Manifest files are a more general category, more often used to describe metadata (a description of a package).

A "dependency file" is a term used for any file that contains dependencies and is used across languages. I haven't found any other term used to classify package.json. If you know of one that is not jargon, let me know.

Reply:
Also, from the link you posted:

> The package.json file is kind of a manifest for your project.

This is not the same as calling it a manifest file.